iOS-Settings/Sources/App/Setting.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Setting.swift
+//  iOS-Settings
+//
+//  Created by Kuanysh al-Khattab Auyelgazy on 08.03.2023.
+//
+
+import UIKit
+
+enum SettingType {
+    case plain
+    case detailed
+    case switcher
+}
+
+struct Setting {
+    let type: SettingType
+    let name: String
+    let detail: String?
+    let iconImage: UIImage
+    let iconColor: UIColor
+
+    init(type: SettingType, name: String, detail: String? = nil, iconImage: UIImage, iconColor: UIColor) {
+        self.type = type
+        self.name = name
+        self.detail = detail
+        self.iconImage = iconImage
+        self.iconColor = iconColor
+    }
+}
+
+extension Setting {
+    static let networkSection = [
+        Setting(type: .switcher, name: "Авиарежим", iconImage: UIImage(systemName: "airplane")!, iconColor: .systemOrange),
+        Setting(type: .detailed, name: "Wi-Fi", detail: "Не подключено",  iconImage: UIImage(systemName: "wifi")!, iconColor: .systemBlue),
+        Setting(type: .detailed, name: "Bluetooth", detail: "Вкл.",  iconImage: UIImage(named: "bluetooth")!, iconColor: .systemBlue),
+        Setting(type: .plain, name: "Сотовая связь",  iconImage: UIImage(named: "cellular")!, iconColor: .systemGreen),
+        Setting(type: .plain, name: "Режим модема",  iconImage: UIImage(systemName: "personalhotspot")!, iconColor: .systemGreen),
+        Setting(type: .switcher, name: "VPN",  iconImage: UIImage(named: "vpn")!, iconColor: .systemGreen),
```

Comment:
Try to not use force unwrap. It can break your app. It is better to use optional value. If value is nil you will handle it according to your design or business needs